docs/openapi.json

```diff
@@ -0,0 +1,2499 @@
+                                "$ref": "#/components/schemas/cnStatistics"
+                              },
+                              {
+                                "type": "object",
+                                "nullable": false,
+                                "properties": {
+                                  "id": {
+                                    "type": "integer",
+                                    "nullable": false
+                                  },
+                                  "kvsets": {
+                                    "type": "array",
+                                    "nullable": false,
+                                    "items": {
+                                      "type": "object",
+                                      "description": "Kvset.",
+                                      "nullable": false,
+                                      "allOf": [
+                                        {
+                                          "$ref": "#/components/schemas/cnStatistics"
+                                        },
+                                        {
+                                          "type": "object",
+                                          "nullable": false,
+                                          "required": [
+                                            "compc",
+                                            "vgroups",
+                                            "rule"
+                                          ],
+                                          "properties": {
+                                            "compc": {
+                                              "type": "integer",
+                                              "nullable": false
+                                            },
+                                            "vgroups": {
+                                              "type": "integer",
+                                              "nullable": false
+                                            },
+                                            "rule": {
+                                              "type": "string",
+                                              "nullable": false,
+                                              "enum": [
+                                                "none",
+                                                "ingest",
+                                                "rspill",
+                                                "rtiny",
+                                                "lbig",
+                                                "lbig1",
+                                                "lgarb",
+                                                "llong",
+                                                "lidxf",
+                                                "lidxp",
+                                                "lidle",
+                                                "lscatf",
```

Comment:
Is there any benefit to using enums here?  There is a cost for sure.  But I don't see how any benefit can justify the cost.